Libraries/PeriphDrivers/Source/ADC/adc_revb.c

```diff
@@ -214,7 +214,7 @@ int MXC_ADC_RevB_StartConversionDMA(mxc_adc_revb_regs_t *adc, mxc_adc_conversion
 
     num_bytes = (req->num_slots + 1) * 4; //Support 8 slots (32 bytes) only. (TODO)
 
-    channel = MXC_DMA_AcquireChannel();
+    channel = req->dma_channel;
```

Comment:
This change will break builds for the other chips which use ADC Rev. B. If we proceed with this implementation these changes will need to be applied to the ME12 and ME18 as well.

Reply:
Yup sure,
I have updated these changes for both ME12 and ME18. 
I have also implemented these changes for MAX78002, since it uses ADC Rev B.